docs/libcurl/opts/CURLOPT_HTTP_VERSION.3

```diff
@@ -30,8 +30,7 @@ CURLcode curl_easy_setopt(CURL *handle, CURLOPT_HTTP_VERSION, long version);
 .SH DESCRIPTION
 Pass \fIversion\fP a long, set to one of the values described below. They ask
 libcurl to use the specific HTTP versions. This is not sensible to do unless
-you have a good reason. You have to set this option if you want to use
-libcurl's HTTP/2 support.
+you have a good reason.
```

Comment:
I think this sentence starting with `This is not sensible...` can be removed as well. I don't think it belongs there.

Reply:
Will do.

Reply:
Done.